README.md

```diff
@@ -48,19 +48,23 @@ Now to the more technical aspects. These are not steps so much as things to keep
 
 ## Asking technical questions:
 
-Take your time, and describe as much of the context as you can manage. You certainly don’t have to assume zero knowledge, but it is really important to think about how you can make it as clear as possible to someone who might not have all the context but may still be able to help. Taking some time to think about how best to ask the question will help you to get the best sort of responses. It’s much more key than you might think, and small efforts to be thoughtful here have big flow-on effects.
+Take your time and describe as much of the context as you can manage. You certainly don’t have to assume zero knowledge, but it is really important to think about how you can make it as clear as possible to someone who might not have all the context but may still be able to help. Taking some time to think about how best to ask the question will help you to get the best sort of responses. It’s much more important than you might think, and small efforts to be thoughtful here have big flow-on effects.
 
 
 ## Code style:
 
 Keep it simple. I sound really boring saying this, but we just want nice simple code that works and that people can understand. Obviously this is within reason, and is also somewhat subjective. But the first thing to consider when writing the code is how easily it will be understood, and how will it make things easier to be build upon. Remember that this code will need to be read by other developers of different skill levels, so just writing code that works is really the baseline.  
```

Comment:
>  I sound really boring saying this, but ....

Some of the sentences are in first person perspective; maybe it might be better to phrase it differently?

```suggestion
Keep it simple. It sounds really boring saying this, but ....
```

Reply:
Great point.